src/spacebro-client.js

```diff
@@ -181,30 +182,29 @@ class SpacebroClient {
   }
 
   sendTo (eventName, to = null, data = {}) {
-    if (this.connected) {
-      data._to = to
-      data._from = this.config.client.name
-      for (let pack of _filterHooks(eventName, this.packers)) {
-        data = pack({eventName, data}) || data
-      }
-      this.socket.emit(eventName, data)
-    } else {
-      this.logger.warn('can\'t emit, not connected.')
+    if (!this.connected) {
+      console.error(`Error: "${this.config.client.name}" is disconnected and cannot emit "${eventName}"`)
```

Comment:
I interpreted this.logger as being intended for 'internal' logging (a message was received, connection started, etc); while "trying to emit from an unconnected socket" is a logic error that should never be silenced.